CONTRIBUTING.md

```diff
@@ -0,0 +1,504 @@
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "firehose:ListDeliveryStreams",
+                "iot:*",
+                "kinesis:ListStreams",
+                "kinesisvideo:ListFragments",
+                "kinesisvideo:ListStreams",
+                "kms:CreateKey",
+                "lambda:ListFunctions",
+                "mobiletargeting:PutEvents",
+                "mobiletargeting:UpdateEndpoint",
+                "polly:DeleteLexicon",
+                "polly:GetSpeechSynthesisTask",
+                "polly:PutLexicon",
+                "polly:StartSpeechSynthesisTask",
+                "polly:SynthesizeSpeech",
+                "s3:*",
+                "sdb:ListDomains",
+                "ses:GetSendQuota",
+                "ses:VerifyEmailIdentity",
+                "sns:GetEndpointAttributes",
+                "sns:ListTopics",
+                "sqs:ListQueues",
+                "textract:AnalyzeDocument",
+                "transcribe:*",
+                "translate:TranslateText"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "VisualEditor3",
+            "Effect": "Allow",
+            "Action": "elasticloadbalancing:ConfigureHealthCheck",
+            "Resource": "arn:aws:elasticloadbalancing:*:*:loadbalancer/*"
+        },
+        {
+            "Sid": "IoTStreamingTest",
+            "Effect": "Allow",
+            "Action": "iam:PassRole",
+            "Resource": "arn:aws:iam::XXXXXXXXXXX:role/OTAIntegTest"
+        }
+    ]
+}
+```
+
+**Policy 2**
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "mobileanalytics:PutEvents",
+                "cognito-sync:*"
+            ],
+            "Resource": [
+                "*"
+            ]
+        }
+    ]
+}
+```
+
+## Workflows
+
+### New Features or Bug Fixes
+
+The AWS SDK for iOS is mostly code generated off of models written in JSON and each time a new feature or new service at AWS launches, these models get updated and thus our SDK also gets updated. Beyond these low-level generated SDK's, we have built some higher level clients that predate the creation of [Amplify](https://github.com/aws-amplify/amplify-ios) (our high-level libraries built on top of the AWS SDK). These higher level clients include:
+
+- `AWSMobileClient`
+- `AWSCognitoAuth`
+- `AWSS3TransferUtility`
+- `AWSLexInteractionKit`
+- `AWSIoTDataManager`
+- `AWSPinpointTargetingClient`
+- `AWSPinpointAnalyticsClient`
+- `AWSPollySynthesizeSpeechURLBuilder`
+- `AWSTranscribeStreamingService`
+- `AWSMobileAnalyticsEventClient`
+
+You could go into any of these files and start adding new endpoints you see in the generated files to these higher level clients or fixing bugs as this is mostly likely where they would occur since it is not code generated code based off of models given to us by the service teams themselves.
```

Comment:
Not sure if I fully understand this sentence. adding endpoints?

Reply:
interfaces

Reply:
endpoints is ambiguous here you are right